.gitignore

```diff
@@ -1,6 +1,7 @@
 .vscode/ipch
 *.code-workspace
 node_modules
+package-lock.json
```

Comment:
As far as I remember, the lockfile should be tracked by git

Reply:
fixed

Reply:
and remove it from the gitignore please

Reply:
I added the package.json file 😂

Reply:
refixed

Reply:
please don't merge now!
I should to add a small thing